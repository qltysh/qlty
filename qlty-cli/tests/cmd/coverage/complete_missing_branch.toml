args = [
  "coverage",
  "complete",
  "--dry-run",
  "--override-commit-sha",
  "2ca1bc45a94e37c8dbae6fd9e19fc069ba64bd67",
  "--override-build-id",
  "123"
]
bin.name = "qlty"
status.code = 99

[env]
remove = ["GITHUB_ACTIONS", "GITHUB_REF_TYPE", "GITHUB_REF_NAME", "GITHUB_HEAD_REF", "CI", "BUILDKITE", "CIRCLECI", "TRAVIS", "GITLAB_CI", "SEMAPHORE"]

[env.add]
QLTY_COVERAGE_TOKEN = "123"
