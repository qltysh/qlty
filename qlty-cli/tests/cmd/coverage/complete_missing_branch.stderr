qlty [..] [..]-[..] ([..] debug [..])
https://qlty.sh/d/coverage

 SETTINGS 

    cwd: [CWD]
    override-build-id: 123
    override-commit-sha: 2ca1bc45a94e37c8dbae6fd9e19fc069ba64bd67

WARNING: The `[[ignore]]` field in qlty.toml is deprecated. Please use `[[exclude]]` or `exclude_patterns` instead.

   ERROR   

 > Unable to determine branch name from the environment.
 > Please provide it using --override-branch
